src/v2/view-builder/views/ov/SelectEnrollmentChannelOktaVerifyView.js

```diff
@@ -16,6 +16,8 @@ const Body = BaseForm.extend({
     channelField.options = _.filter(channelField?.options, (option) =>
       option.value !== this.options.appState.get('currentAuthenticator')?.contextualData?.selectedChannel);
     channelField.value = channelField.options[0]?.value;
+    channelField.sublabel = null;
```

Comment:
Please advice, why we need to clean this ?

Reply:
Currently, there is an "optional" text that shows up on the 'More options' page. There is a requirement to have it removed.